### DISCUSSION TITLE
Import more then "desc" from GPX (?)

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#13782] [Q&A]
Hello!

I have GPX-Files with detailled informations in different tags: name, desc, cmt, link.

OSMAnd only shows the "desc"-tag. Is it possible, to import/display all tags?

GPX:
![grafik](https://user-images.githubusercontent.com/62500258/154845402-213d1ec8-3d8d-4c56-8ada-deee35365272.png)

OSMAnd:
![grafik](https://user-images.githubusercontent.com/62500258/154845370-2addc720-7f94-4163-a6c5-a586fadda3c8.png)

same example in LocusMaps:
![grafik](https://user-images.githubusercontent.com/62500258/154845386-c37cb6a9-56a5-426d-a84a-3e2e63349851.png)

**Comment by vshcherb** [comment]
I think it's a context menu of favorite group not a displayed gpx. Do you have it to test?

**Reply by ThomasMeister123** [reply]
I dont know what you mean. I selected the imported POI (from GPX) on the map. On the screenshot you see the Imported properties. There is only the DESC. LocusMaps has imported ALL information

**Reply by vshcherb** [reply]
I think it's worth opening an issue to keep track of it.

**Reply by Lee-Carre** [reply]
> opening an issue

#13811
